pkg/storegateway/storegatewaypb/rpc.proto

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
```

Comment:
This needs to be changed to Mimir, but we have to reference Thanos. See for example what we do in other files imported from Thanos, e.g.:

```
// SPDX-License-Identifier: AGPL-3.0-only
// Provenance-includes-location: https://github.com/thanos-io/thanos/blob/main/pkg/store/bucket.go
// Provenance-includes-license: Apache-2.0
// Provenance-includes-copyright: The Thanos Authors.
```